components/LinearGradient/LinearGradient.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled } from "@mui/system";
```

Comment:
shouldn't it be styled form styled-components? or is it the same? I am not familiar with raw MUI

Reply:
The `styled` from `@mui/system`  is the same as from `styled-components`. 
https://mui.com/system/styled/#what-problems-does-it-solve